src/messages/web-compat.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "WebCompatMessages",
+  "description": "Requests, Notifications and Subscriptions from the Web Compat feature",
+  "additionalProperties": false,
+  "properties": {
+    "notifications": {},
+    "requests": {
+      "oneOf": [
+        {
+          "$ref": "./web-compat/web-share.json"
+        }
+      ]
+    },
+    "subscriptions": {}
```

Comment:
for `notifications`, `requests` and `subscriptions`, we can use `oneOf` in JSON schema, which gives the following output when there is multiple items

```
interface Messages {
   requests: Msg1 | Msg2 | Msg3,
}
```